src/test/java/Pages/homePage.java

```diff
@@ -0,0 +1,110 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class homePage extends basePage {
+    WebDriver driver;
+    static WebDriverWait wait;
+    String playlistName = "HW22";
+   By userAvatarIcon = By.cssSelector("img.avatar");
+     By newPlaylistIcon = By.cssSelector("i[title='Create a new playlist']");
+   By newPlaylist = By.cssSelector("li[data-testid='playlist-context-menu-create-simple']");
+     By nameField = By.cssSelector("input[name='name']");
+   By searchSong = By.cssSelector("[type='search'");
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)